admin-front/src/data/dataProvider.ts

```diff
@@ -49,31 +49,33 @@ const dataProvider = (
     const rangeEnd = perPage;
 
     let query = {};
-    if (resource === 'analytics/sales/priceVolume/snapshot') {
-      query = {
-        sort: JSON.stringify([
-          field.startsWith('attributes.')
-            ? field.slice('attributes.'.length)
-            : field,
-          order.toLowerCase(),
-        ]),
-        range: JSON.stringify([rangeStart, rangeEnd]),
-        [Object.keys(params.filter)[0]]: params.filter.resolution,
-      };
-    } else {
-      query = {
-        sort: JSON.stringify([
-          field.startsWith('attributes.')
-            ? field.slice('attributes.'.length)
-            : field,
-          order.toLowerCase(),
-        ]),
-        range: JSON.stringify([rangeStart, rangeEnd]),
-        filters:
-          Object.keys(params.filter).length === 0
-            ? undefined
-            : JSON.stringify(params.filter),
-      };
+    switch (resource) {
+      case 'analytics/sales/priceVolume/snapshot':
+        query = {
+          sort: JSON.stringify([
+            field.startsWith('attributes.')
+              ? field.slice('attributes.'.length)
+              : field,
+            order.toLowerCase(),
+          ]),
+          range: JSON.stringify([rangeStart, rangeEnd]),
```

Comment:
Not from this PR itself (this code was already there before), but `snapshot` doesn't have `range` or `sort` logic. Or maybe I am missing something, do you know more about this? If not I'll have a further look at this code to see whether this actually makes sense.

Reply:
Yes, no range or sort logic. Will remove it in the next PR as I do not know if it has any effect on the client code if I remove it from the data provider.